.github/workflows/build_windows_packages.yml

```diff
@@ -16,15 +16,30 @@ on:
 jobs:
   build_windows_packages:
     name: Build Windows Packages
-    runs-on: windows-2022  # TODO(#36): move to cluster of CPU builders like `azure-windows-scale-nod-ai`
+    runs-on: azure-windows-scale-nod-ai
+    env:
+      BASE_BUILD_DIR_POWERSHELL: C:\mnt\azure\b
     defaults:
       run:
         shell: bash
     strategy:
       fail-fast: true
     steps:
+      - name: "Create build dir"
+        shell: powershell
+        run: |
+          $currentTime = (Get-Date).ToString('HHmmss')
+          $buildDir = "$env:BASE_BUILD_DIR_POWERSHELL\$currentTime"
+          echo "BUILD_DIR_POWERSHELL=$buildDir" >> $env:GITHUB_ENV
+          mkdir "$buildDir"
+          Write-Host "Generated Build Directory: $buildDir"
+          $bashBuildDir = $buildDir -replace '\\', '/' -replace '^C:', '/c'
+          echo  "BUILD_DIR_BASH=$bashBuildDir" >> $env:GITHUB_ENV
+          Write-Host "Converted Build Directory For Bash: $bashBuildDir"
```

Comment:
Not happy with this hack spreading across workflows / repositories / organizations. This should really be part of the runner configuration itself, if it is truly needed. I can look the other way again this time, but this hack has already lived too long.